web/themes/custom/pineapple/sass/base/forms/_contact-form.scss

```diff
@@ -0,0 +1,3 @@
+.contact-form {
```

Comment:
Does this still need to be here if you removed the class?

Reply:
no